### DISCUSSION TITLE
Daily visitors total

### DISCUSSION THREAD

**Discussion by SamuelB-ACA** [allinurl/goaccess#2372] [Q&A]
Hi,
That's probably obvious but I can't see where I can get the total daily visitors.
My stats look like this

![image](https://user-images.githubusercontent.com/87703566/188456861-9ae383dc-6034-4d5f-9c08-ab066c95493c.png)

But I can't find the daily total

**Comment by allinurl** [comment]
Great observation! If you are using `--date-spec`, currently goaccess does not show the total. I can look into adding this. Where do you think it would be the best place to show this? Thanks

**Reply by SamuelB-ACA** [reply]
Hard to say. Maybe at least in the popup
![image](https://user-images.githubusercontent.com/87703566/188813570-0375c42c-f50f-4de2-bff0-b12cebffa39b.png)
Or to keep the design neat, a link at the right of pagination that would open a popin with the datas.
I'm really bad at design and ergonomics :)
